test/sanity/validate-modules/schema.py

```diff
@@ -17,29 +17,33 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from voluptuous import ALLOW_EXTRA, Any, Required, Schema
+from ansible.compat.six import string_types
+# Because string_types from six is tuple mostly used in isinstance()
+# function and we need just class name for schema definition
+string_types = string_types[0]
```

Comment:
Ok, I am gonna change it. I'll have to add `Any()` on lines where only `*string_types` will be used and change the order of arguments because the one with the star has to be the last one.